node/cli/src/command.rs

```diff
@@ -205,6 +205,11 @@ fn extract_genesis_wasm(chain_spec: &Box<dyn sc_service::ChainSpec>) -> Result<V
 /// Parse command line arguments into service configuration.
 pub fn run() -> Result<()> {
 	let cli = Cli::from_args();
+	if (cli.run.ethapi.contains(&EthApi::Debug) || cli.run.ethapi.contains(&EthApi::Trace))
```

Comment:
We could make it better by printing a warning that the node is not compiled with tracing in that case ?

Reply:
> It would be better to include the --ethapi option only when we compile with tracing

Imo we want that option to be always available, and guide the user like above on _how_ to use it (compiling with the feature).

> We could make it better by printing a warning

A warning instead an error? Above if you include the option and there's no `evm-tracing` feature, just errors with `--ethapi only available on evm-tracing feature.`

Reply:
Ok, indeed it is better if there is an error message